src/main/java/com/ning/http/util/UTF8UrlEncoder.java

```diff
@@ -55,25 +208,57 @@ public static String encode(String input) {
         return sb.toString();
     }
 
+    public static StringBuilder encodeAppendScheme(StringBuilder sb, String scheme) {
```

Comment:
the existing method was `appendEncoded`. Was this supposed to be `appendEncodedScheme`? (same question for all of the following `encodeAppend*` methods.